lib/editor/editor.dart

```diff
@@ -178,50 +180,53 @@ class EditorState extends State<Editor> {
       isLoading = false;
     });
 
-    TextEditingControllerIDE.language = widget.language;
+    TextEditingControllerIDE.language = widget.defaultLanguage;
   }
 
-  handleRegionFields(FileIDE file) async {
+  handleRegionFields() async {
     SharedPreferences prefs = await SharedPreferences.getInstance();
+    EditorRegionOptions? region = widget.options.regionOptions;
+    String path = widget.path;
+    String fileContent = widget.defaultValue;
 
-    if (file.hasRegion) {
-      int regionStart = file.region.start!;
-      int regionEnd = file.region.end!;
+    if (widget.options.regionOptions != null) {
```

Comment:
You're storing `regionOptions` in `region` variable. Use that as then you won't have to use the not-null assertion operator in the line below